### PR TITLE
Modify project init task

### PR DESCRIPTION
**Pull request by udit-gulati**
Something similar to what create-react-app does:

```bash
Success! Created helloreact at /home/uditgulati/helloreact
Inside that directory, you can run several commands:

  yarn start
    Starts the development server.

  yarn build
    Bundles the app into static files for production.

  yarn test
    Starts the test runner.

  yarn eject
    Removes this tool and copies build dependencies, configuration files
    and scripts into the app directory. If you do this, you can’t go back!

We suggest that you begin by typing:

  cd helloreact
  yarn start

Happy hacking!
```

### CODE REVIEW

File: packages/polar/src/internal/cli/project-creation.ts [hunk 1]
```diff
@@ -47,15 +49,21 @@ function copySampleProject (location: string): void {
   });
 }
 
-function printSuggestedCommands (): void {
+function printSuggestedCommands (projectName: string): void {
+  const currDir = process.cwd();
+  const projectPath = path.join(currDir, projectName);
+  console.log(`Success! Created project at ${chalk.greenBright(projectPath)}.`);
+  // console.log(`Inside that directory, you can run several commands:`);
```
Comment (coco-sha) [inline packages/polar/src/internal/cli/project-creation.ts:56]: can we remove this line or uncomment it ?

Comment (udit-gulati) [reply packages/polar/src/internal/cli/project-creation.ts:56]: This I've left if we later we want to add description of all commands later. But if not, I can remove it.

**Comment by udit-gulati** [comment]
Now doing `polar init helloproject` would create a directory with name `helloproject` which'll have project contents in it and user has to provide the projectName in the command.

File: packages/polar/src/internal/cli/project-creation.ts (hunk 1 above)
Comment (coco-sha) [reply packages/polar/src/internal/cli/project-creation.ts:56]: ok, Please add a TODO: ... here

**Review by coco-sha — APPROVED** [review]
Please update the main readme.md with updated command description

File: packages/polar/src/lib/deploy/compress.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { cmpStr } from "../../internal/util/strings";
+import type { PolarRuntimeEnvironment } from "../../types";
+
+export async function compress (
+  networkName: string[],
+  env: PolarRuntimeEnvironment
+): Promise<void> {
+  const paths = readdirSync(CONTRACTS_DIR);
+  if (paths.includes("Cargo.toml")) { // Only one contract in the contracts dir and compile in contracts dir only
+    const imageVersion = `0.11.5`;
```
Comment (coco-sha) [inline packages/polar/src/lib/deploy/compress.ts:22]: could you please move this to CONSTANTS ?

File: packages/polar/src/lib/deploy/compress.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    const fileContents = fs.createReadStream(path.join(CONTRACTS_DIR, "contract.wasm.gz"));
+    const writeStream = fs.createWriteStream(path.join(CONTRACTS_OUT_DIR, "contract.wasm"));
+    const unzip = zlib.createGunzip();
+
+    fileContents.pipe(unzip).pipe(writeStream);
+    fs.unlinkSync(path.join(CONTRACTS_DIR, "contract.wasm.gz"));
+    console.log(`Created file ${path.join(CONTRACTS_OUT_DIR, "contract.wasm")}`);
+  } else { // Multiple contracts and each should be compiled by going inside each of them
+    const imageVersion = `0.11.5`;
```
Comment (coco-sha) [inline packages/polar/src/lib/deploy/compress.ts:39]: same

**Pull request merged**